frontend/13f-app/src/components/CompanyFilingList.css

```
@@ -0,0 +1,32 @@
+#filings-list {
+    font-family: Arial, Helvetica, sans-serif;
+    border-collapse: collapse;
+    width: 80%;
+    margin-top: 25px;
+    margin-left: 25px;
```

Use the [margin shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/margin)

```suggestion
    margin: 25px 0 0 25px;
```